### PR TITLE
perlPackages.Mojolicious: 8.71 -> 9.10

### PR DESCRIPTION
**Pull request by zakame**
###### Motivation for this change

https://metacpan.org/release/SRI/Mojolicious-9.10

As this is a major release, this also includes some updates of other modules that depend on this, as well as introduces the new `perlPackages.MojoIOLoopDelay` dist that spins off the deprecated `Mojo::IOLoop::Delay` module from the main Mojolicious release.

Another change is ~~marking `perlPackages.OpenAPIClient` as broken due to it needing a removed module from `JSON::Validator`~~ updating `perlPackages.OpenAPIClient` and `perlPackages.JSONValidator` to account for their new Mojolicious feature dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114065` at 6166466a run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>perl530Packages.OpenAPIClient</li>
    <li>perl532Packages.OpenAPIClient</li>
  </ul>
</details>
<details>
  <summary>48 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/d8b2489bc9cff9c6bde9fc53d3dcdc13">abcde</a></li></li>
    <li><a href="https://gist.github.com/6a6212586565f8801b4b91c0877a4aae">convos</a></li></li>
    <li><a href="https://gist.github.com/bc50cce0e286281396dbe2b8cafc73af">perl530Packages.JSONValidator</a></li></li>
    <li><a href="https://gist.github.com/dd72888da4d24bd05132c5e174a36ace">perl530Packages.LinkEmbedder</a></li></li>
    <li><a href="https://gist.github.com/a4519f3d7c3f6a426de636df3b6a5c7b">perl530Packages.Minion</a></li></li>
    <li><a href="https://gist.github.com/0621bb8e784375806d56e6cac6535c59">perl530Packages.MinionBackendSQLite</a></li></li>
    <li><a href="https://gist.github.com/7827b7d5bca126ea25c58b19a0ae865e">perl530Packages.MinionBackendmysql</a></li></li>
    <li><a href="https://gist.github.com/9d513570fb31d066e183995ed9c32fc4">perl530Packages.MojoIOLoopDelay</a></li></li>
    <li><a href="https://gist.github.com/45bb41eb6505932eda95f0f971a000f1">perl530Packages.MojoIOLoopForkCall</a></li></li>
    <li><a href="https://gist.github.com/e175be8489824d4bdbc111f40441fc38">perl530Packages.MojoJWT</a></li></li>
    <li><a href="https://gist.github.com/f3b9514fb19f68bebf88b5a5eadfe9bf">perl530Packages.MojoPg</a></li></li>
    <li><a href="https://gist.github.com/530ec6d6d32bf0582a76ddce1d146702">perl530Packages.MojoRedis</a></li></li>
    <li><a href="https://gist.github.com/25c0b19ddc8f54eeb432b83ad86668d2">perl530Packages.MojoSAML</a></li></li>
    <li><a href="https://gist.github.com/f20368d248d445157ab7f47eb6be75e6">perl530Packages.MojoSQLite</a></li></li>
    <li><a href="https://gist.github.com/a6e9423c5c83f2d5ff402afde3e32069">perl530Packages.MojoUserAgentCached</a></li></li>
    <li><a href="https://gist.github.com/e93daa617e1c3645526f7ce44061a6b2">perl530Packages.Mojolicious</a></li></li>
    <li><a href="https://gist.github.com/0d542d80cfd2f2819516ad440a7c5724">perl530Packages.MojoliciousPluginAssetPack</a></li></li>
    <li><a href="https://gist.github.com/a66ce54fadcdde4b16737d6782449513">perl530Packages.MojoliciousPluginGravatar</a></li></li>
    <li><a href="https://gist.github.com/ce4f6a56da063d3fe8b12816f8d97a1e">perl530Packages.MojoliciousPluginMail</a></li></li>
    <li><a href="https://gist.github.com/ca944eef737455ad8209f4179b9dca2b">perl530Packages.MojoliciousPluginOpenAPI</a></li></li>
    <li><a href="https://gist.github.com/18aff6693e72a681b88321bd7862bf55">perl530Packages.MojoliciousPluginStatus</a></li></li>
    <li><a href="https://gist.github.com/0a3d284e4c5578a918dfe79f2028a487">perl530Packages.MojoliciousPluginTextExceptions</a></li></li>
    <li><a href="https://gist.github.com/c712a0890a461f8ea398c392770f6973">perl530Packages.MojoliciousPluginWebpack</a></li></li>
    <li><a href="https://gist.github.com/4fc764ee42f6a61e79415cc30d78a105">perl530Packages.Mojomysql</a></li></li>
    <li><a href="https://gist.github.com/abd9cecf4af1e3cc02f7ad4a462c08ea">perl530Packages.MusicBrainz</a></li></li>
    <li><a href="https://gist.github.com/61ccee27d1e023c50a3d8ca30be124dc">perl532Packages.JSONValidator</a></li></li>
    <li><a href="https://gist.github.com/1d9ccf36932dd96d203599e947eeb44c">perl532Packages.LinkEmbedder</a></li></li>
    <li><a href="https://gist.github.com/a8cc769e44c85b5debb4ac12511c207c">perl532Packages.Minion</a></li></li>
    <li><a href="https://gist.github.com/9dea77b750edea82412955c069ac26dc">perl532Packages.MinionBackendSQLite</a></li></li>
    <li><a href="https://gist.github.com/7cfe892f6e77ad0a06bf6a78b9def764">perl532Packages.MinionBackendmysql</a></li></li>
    <li><a href="https://gist.github.com/29195c22af5fb55185bf4eb160827a9f">perl532Packages.MojoIOLoopDelay</a></li></li>
    <li><a href="https://gist.github.com/4e2e5838563d969688ee5363a22a326b">perl532Packages.MojoIOLoopForkCall</a></li></li>
    <li><a href="https://gist.github.com/0486abff932944bdc4cae478d4d27c45">perl532Packages.MojoJWT</a></li></li>
    <li><a href="https://gist.github.com/f753a95944fdadfb862540cd382f0720">perl532Packages.MojoPg</a></li></li>
    <li><a href="https://gist.github.com/404638e34af07a643ce556b1b41e0e13">perl532Packages.MojoRedis</a></li></li>
    <li><a href="https://gist.github.com/41f3de561a63840dccdcc85fe9d4fd96">perl532Packages.MojoSAML</a></li></li>
    <li><a href="https://gist.github.com/190ca52fea21fb4ae847dfd8281dccbe">perl532Packages.MojoSQLite</a></li></li>
    <li><a href="https://gist.github.com/f1d58e1e2324586a9e6de57c7d2e775d">perl532Packages.MojoUserAgentCached</a></li></li>
    <li><a href="https://gist.github.com/698eac2372d2ba876e128a03c74a0757">perl532Packages.Mojolicious</a></li></li>
    <li><a href="https://gist.github.com/63cf6a8c39ef9faa25d1b01cc829d4a7">perl532Packages.MojoliciousPluginAssetPack</a></li></li>
    <li><a href="https://gist.github.com/e307a44c749a5e147e81ab2a112f143b">perl532Packages.MojoliciousPluginGravatar</a></li></li>
    <li><a href="https://gist.github.com/19c11b76da5d80c94bae8d2991ed1982">perl532Packages.MojoliciousPluginMail</a></li></li>
    <li><a href="https://gist.github.com/2ebb31ad1f1205bd36c4137a0e97201d">perl532Packages.MojoliciousPluginOpenAPI</a></li></li>
    <li><a href="https://gist.github.com/0483569e256dffd27bb6ac5f7bb96f1b">perl532Packages.MojoliciousPluginStatus</a></li></li>
    <li><a href="https://gist.github.com/fb8c2a26c6d9f7d007eefc5ad1061e49">perl532Packages.MojoliciousPluginTextExceptions</a></li></li>
    <li><a href="https://gist.github.com/39e6a9fe1dab1bdb5b3bc00815d30e4b">perl532Packages.MojoliciousPluginWebpack</a></li></li>
    <li><a href="https://gist.github.com/e94cde0722f22e918c11d074eb78757a">perl532Packages.Mojomysql</a></li></li>
    <li><a href="https://gist.github.com/866ef5d39e0719d2dfa8785c6f9f3aee">perl532Packages.MusicBrainz</a></li></li>
  </ul>
</details>

**Comment by zakame** [comment]
`perlPackages.OpenAPIClient` will soon get an updated release, so maybe holding this off a bit...

**Comment by SuperSandro2000** [comment]
Fails to build on darwin:

```
these derivations will be built:
  /nix/store/03fya32xj5xabcx1vm3jgfln9f8wdnqa-perl5.30.3-PerlIO-Layers-0.012.drv
  /nix/store/jgp1awb3pig04nv0x2l95sf2rvq8yhh6-perl5.30.3-File-Map-0.67.drv
  /nix/store/ccfalyrz19fkwzssv4hskz3f416c486m-perl5.30.3-Mojolicious-Plugin-Status-1.12.drv
building '/nix/store/03fya32xj5xabcx1vm3jgfln9f8wdnqa-perl5.30.3-PerlIO-Layers-0.012.drv'...
unpacking sources
unpacking source archive /nix/store/qxiwa8vhg4lvfbmsqvf2fvabsdn00rql-PerlIO-Layers-0.012.tar.gz
source root is PerlIO-Layers-0.012
setting SOURCE_DATE_EPOCH to timestamp 1581985836 of file PerlIO-Layers-0.012/xt/author/pod-syntax.t
patching sources
configuring
patching ./t/10-basics.t...
patching ./xt/author/pod-syntax.t...
no configure script, doing nothing
building
ld: unknown option: -mmacosx-version-min=10.12
Warning: ExtUtils::CBuilder not installed or no compiler detected
Proceeding with configuration, but compilation may fail during Build

Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'PerlIO-Layers' version '0.012'
Building PerlIO-Layers
Error: no compiler detected to compile 'lib/PerlIO/Layers.c'.  Aborting
builder for '/nix/store/03fya32xj5xabcx1vm3jgfln9f8wdnqa-perl5.30.3-PerlIO-Layers-0.012.drv' failed with exit code 255
cannot build derivation '/nix/store/jgp1awb3pig04nv0x2l95sf2rvq8yhh6-perl5.30.3-File-Map-0.67.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/ccfalyrz19fkwzssv4hskz3f416c486m-perl5.30.3-Mojolicious-Plugin-Status-1.12.drv': 1 dependencies couldn't be built
error: build of '/nix/store/ccfalyrz19fkwzssv4hskz3f416c486m-perl5.30.3-Mojolicious-Plugin-Status-1.12.drv' failed
```

**Comment by zakame** [comment]
@SuperSandro2000 thanks, looks like a general `ld` issue on macOS though?  `perlPackages.perlIOLayers` wasn't changed so not sure what happened there.

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 thanks, looks like a general `ld` issue on macOS though? `perlPackages.perlIOLayers` wasn't changed so not sure what happened there.

Maybe it mixes up clang and ld. I have seen that flag multiple in combination with clang but not ld. Making the package broken on darwin would also be okay for me.

**Comment by zakame** [comment]
> Maybe it mixes up clang and ld. I have seen that flag multiple in combination with clang but not ld. Making the package broken on darwin would also be okay for me.

Hmm, let me check further :muscle: could go for a separate PR for `perlPackages.PerlIOLayers` in that case, to go ahead of this one.

**Comment by zakame** [comment]
> Maybe it mixes up clang and ld. I have seen that flag multiple in combination with clang but not ld. Making the package broken on darwin would also be okay for me.

@stigtsp @SuperSandro2000 would a `perlPreHook = lib.optionalString stdenv.isDarwin "export LD=$CC"` also work for `perlPackages.PerlIOLayers`?  I realise the `-mmacosx-version-min=10.12` is meant for the compiler, not linker.

**Comment by SuperSandro2000** [comment]
> @stigtsp @SuperSandro2000 would a `perlPreHook = lib.optionalString stdenv.isDarwin "export LD=$CC"` also work for `perlPackages.PerlIOLayers`? I realise the `-mmacosx-version-min=10.12` is meant for the compiler, not linker.

Can we set it for both Linux and Darwin? Not sure how perlPackages works but I could try to build it.

**Comment by zakame** [comment]
> > @stigtsp @SuperSandro2000 would a `perlPreHook = lib.optionalString stdenv.isDarwin "export LD=$CC"` also work for `perlPackages.PerlIOLayers`? I realise the `-mmacosx-version-min=10.12` is meant for the compiler, not linker.
> 
> Can we set it for both Linux and Darwin? Not sure how perlPackages works but I could try to build it.

Yes, it should just be a `perlPreHook = "export LD=$CC";` then.  I think this is fine since the actual source of the error comes from [`ExtUtils::CBuilder`](https://metacpan.org/source/AMBS/ExtUtils-CBuilder-0.280236/lib%2FExtUtils%2FCBuilder%2FPlatform%2Fdarwin.pm) when it inspects the perl `Config.pm` for applicable flags.

Let me spin that change off into a separate PR since that will rebuild a few more packages unrelated to Mojo (like `perlPackages.PDL` which is somewhat big...)

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 114065` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
50 packages built:
</summary>
<ul>
<li>abcde</li>
<li>convos</li>
<li>perl530Packages.JSONValidator</li>
<li>perl530Packages.LinkEmbedder</li>
<li>perl530Packages.Minion</li>
<li>perl530Packages.MinionBackendSQLite</li>
<li>perl530Packages.MinionBackendmysql</li>
<li>perl530Packages.MojoIOLoopDelay</li>
<li>perl530Packages.MojoIOLoopForkCall</li>
<li>perl530Packages.MojoJWT</li>
<li>perl530Packages.MojoPg</li>
<li>perl530Packages.MojoRedis</li>
<li>perl530Packages.MojoSAML</li>
<li>perl530Packages.MojoSQLite</li>
<li>perl530Packages.MojoUserAgentCached</li>
<li>perl530Packages.Mojolicious</li>
<li>perl530Packages.MojoliciousPluginAssetPack</li>
<li>perl530Packages.MojoliciousPluginGravatar</li>
<li>perl530Packages.MojoliciousPluginMail</li>
<li>perl530Packages.MojoliciousPluginOpenAPI</li>
<li>perl530Packages.MojoliciousPluginStatus</li>
<li>perl530Packages.MojoliciousPluginTextExceptions</li>
<li>perl530Packages.MojoliciousPluginWebpack</li>
<li>perl530Packages.Mojomysql</li>
<li>perl530Packages.MusicBrainz</li>
<li>perl530Packages.OpenAPIClient</li>
<li>perl532Packages.JSONValidator</li>
<li>perl532Packages.LinkEmbedder</li>
<li>perl532Packages.Minion</li>
<li>perl532Packages.MinionBackendSQLite</li>
<li>perl532Packages.MinionBackendmysql</li>
<li>perl532Packages.MojoIOLoopDelay</li>
<li>perl532Packages.MojoIOLoopForkCall</li>
<li>perl532Packages.MojoJWT</li>
<li>perl532Packages.MojoPg</li>
<li>perl532Packages.MojoRedis</li>
<li>perl532Packages.MojoSAML</li>
<li>perl532Packages.MojoSQLite</li>
<li>perl532Packages.MojoUserAgentCached</li>
<li>perl532Packages.Mojolicious</li>
<li>perl532Packages.MojoliciousPluginAssetPack</li>
<li>perl532Packages.MojoliciousPluginGravatar</li>
<li>perl532Packages.MojoliciousPluginMail</li>
<li>perl532Packages.MojoliciousPluginOpenAPI</li>
<li>perl532Packages.MojoliciousPluginStatus</li>
<li>perl532Packages.MojoliciousPluginTextExceptions</li>
<li>perl532Packages.MojoliciousPluginWebpack</li>
<li>perl532Packages.Mojomysql</li>
<li>perl532Packages.MusicBrainz</li>
<li>perl532Packages.OpenAPIClient</li>
</ul>
</details>

File: pkgs/top-level/perl-packages.nix
Comment (SuperSandro2000) [inline pkgs/top-level/perl-packages.nix:13724]: homepage and maintainers is missing.

File: pkgs/top-level/perl-packages.nix
Comment (SuperSandro2000) [inline pkgs/top-level/perl-packages.nix:13743]: homepage and maintainers is missing.

File: pkgs/top-level/perl-packages.nix
Comment (zakame) [reply pkgs/top-level/perl-packages.nix:13729]: Thanks, added.

**Comment by zakame** [comment]
Now updated to include Mojolicious-9.02.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 114065` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
50 packages built:
</summary>
<ul>
<li>abcde</li>
<li>convos</li>
<li>perl530Packages.JSONValidator</li>
<li>perl530Packages.LinkEmbedder</li>
<li>perl530Packages.Minion</li>
<li>perl530Packages.MinionBackendSQLite</li>
<li>perl530Packages.MinionBackendmysql</li>
<li>perl530Packages.MojoIOLoopDelay</li>
<li>perl530Packages.MojoIOLoopForkCall</li>
<li>perl530Packages.MojoJWT</li>
<li>perl530Packages.MojoPg</li>
<li>perl530Packages.MojoRedis</li>
<li>perl530Packages.MojoSAML</li>
<li>perl530Packages.MojoSQLite</li>
<li>perl530Packages.MojoUserAgentCached</li>
<li>perl530Packages.Mojolicious</li>
<li>perl530Packages.MojoliciousPluginAssetPack</li>
<li>perl530Packages.MojoliciousPluginGravatar</li>
<li>perl530Packages.MojoliciousPluginMail</li>
<li>perl530Packages.MojoliciousPluginOpenAPI</li>
<li>perl530Packages.MojoliciousPluginStatus</li>
<li>perl530Packages.MojoliciousPluginTextExceptions</li>
<li>perl530Packages.MojoliciousPluginWebpack</li>
<li>perl530Packages.Mojomysql</li>
<li>perl530Packages.MusicBrainz</li>
<li>perl530Packages.OpenAPIClient</li>
<li>perl532Packages.JSONValidator</li>
<li>perl532Packages.LinkEmbedder</li>
<li>perl532Packages.Minion</li>
<li>perl532Packages.MinionBackendSQLite</li>
<li>perl532Packages.MinionBackendmysql</li>
<li>perl532Packages.MojoIOLoopDelay</li>
<li>perl532Packages.MojoIOLoopForkCall</li>
<li>perl532Packages.MojoJWT</li>
<li>perl532Packages.MojoPg</li>
<li>perl532Packages.MojoRedis</li>
<li>perl532Packages.MojoSAML</li>
<li>perl532Packages.MojoSQLite</li>
<li>perl532Packages.MojoUserAgentCached</li>
<li>perl532Packages.Mojolicious</li>
<li>perl532Packages.MojoliciousPluginAssetPack</li>
<li>perl532Packages.MojoliciousPluginGravatar</li>
<li>perl532Packages.MojoliciousPluginMail</li>
<li>perl532Packages.MojoliciousPluginOpenAPI</li>
<li>perl532Packages.MojoliciousPluginStatus</li>
<li>perl532Packages.MojoliciousPluginTextExceptions</li>
<li>perl532Packages.MojoliciousPluginWebpack</li>
<li>perl532Packages.Mojomysql</li>
<li>perl532Packages.MusicBrainz</li>
<li>perl532Packages.OpenAPIClient</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
```
Built '/nix/store/yln548j0inw67kj2pdvwcpiis3k5p40j-perl5.32.1-PerlIO-Layers-0.012.drv' in 2.2 s.                                                                                            Built '/nix/store/1qc4inkvn7icnky8lz683ahqzkidhs1g-perl5.32.1-WebService-MusicBrainz-1.0.5.drv' in 1.4 s.                                                                                   error: --- Error ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- nixbuilder for '/nix/store/138zqxa7fl3lch9nx7kv4lfjmznvm5mp-perl5.32.1-File-Map-0.67.drv' failed with exit code 255; last 10 log lines:                                                          no configure script, doing nothing                                                                                                                                                          building                                                                                                                                                                                    ld: unknown option: -mmacosx-version-min=10.12                                                                                                                                              Warning: ExtUtils::CBuilder not installed or no compiler detected                                                                                                                           Proceeding with configuration, but compilation may fail during Build                                                                                                                                                                                                                                                                                                                    Created MYMETA.yml and MYMETA.json                                                                                                                                                          Creating new 'Build' script for 'File-Map' version '0.67'                                                                                                                                   Building File-Map                                                                                                                                                                           Error: no compiler detected to compile 'lib/File/Map.c'.  Aborting                                                                                                                        error: --- Error ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- nix1 dependencies of derivation '/nix/store/q1z2wpfqbcmqgjhvwjv9s1n9aqcwnqxl-perl5.32.1-Mojolicious-Plugin-Status-1.12.drv' failed to build                                                    Built '/nix/store/cifk74avjswh5k67w4h3kdgjkccswv5p-source.drv' in 2.3 s.                                                                                                                    Built '/nix/store/srpljvd46w6icidjfy246v2xrj2s9xjn-perl5.32.1-Mojolicious-Plugin-OpenAPI-4.00.drv' in 35.5 s.                                                                               Built '/nix/store/2wdz5ybybqdxzl6hx3l8clplljn4r9ig-perl5.32.1-OpenAPI-Client-1.00.drv' in 8.8 s.                                                                                            Built '/nix/store/3nmcd6frhnxldpn7wj0bxqbrbaas7kbh-perl5.32.1-convos-6.00.drv' in 55.3 s.                                                                                                   error: --- Error ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- nix2 dependencies of derivation '/nix/store/0b7vxli9j8c6cwwcld0jvxbnjnp44f3s-review-shell.drv' failed to build
```


![image](https://user-images.githubusercontent.com/7258858/110457643-273da580-80cb-11eb-83fc-c57c16dfdc82.png)

**Comment by zakame** [comment]
@SuperSandro2000 thanks, sounds like `perlPackages.FileMap` needs the same `export LD=$CC` prePerlHook then.

**Comment by zakame** [comment]
This PR's open so long now that I've updated Mojolicious to *9.10*.

File: pkgs/applications/networking/irc/convos/default.nix
Comment (stigtsp) [inline pkgs/applications/networking/irc/convos/default.nix:9]:
```suggestion
  version = "6.06";
```

File: pkgs/applications/networking/irc/convos/default.nix
Comment (stigtsp) [inline pkgs/applications/networking/irc/convos/default.nix:15]:
```suggestion
    sha256 = "0b3c8hj9cjmpzy9k949vdv1y3v7b94nh0mq15rcv3ax0sj3gd0qr";
```

File: pkgs/top-level/perl-packages.nix
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix:13543]:
```suggestion
    version = "2.13";
```

File: pkgs/top-level/perl-packages.nix
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix:13546]:
```suggestion
      url = "mirror://cpan/authors/id/S/SR/SRI/Mojolicious-Plugin-AssetPack-2.13.tar.gz"";
      sha256 = "1254yy70c7wv3p64pjyxc2h1p9czs65jm6lzl42qmn1x19i8fggj";
```

**Review by stigtsp — COMMENTED** [review]
Needs some more bumps, but other than that: looks good to me.

File: pkgs/applications/networking/irc/convos/default.nix
Comment (zakame) [reply pkgs/applications/networking/irc/convos/default.nix]: Thanks, updated!

**Comment by stigtsp** [comment]
@GrahamcOfBorg build perlPackages.Mojolicious convos perlPackages.MojoIOLoopDelay perlPackages.MojoIOLoopForkCall perlPackages.MojoUserAgentCached perlPackages.MinionBackendSQLite perlPackages.MojoliciousPluginAssetPack perlPackages.MojoSQLite perlPackages.JSONValidator perlPackages.MojoliciousPluginOpenAPI perlPackages.OpenAPIClient perlPackages.LinkEmbedder

**Review by stigtsp — APPROVED** [review]
LGTM

Result of `nixpkgs-review pr 114065` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>50 packages built:</summary>
  <ul>
    <li>abcde</li>
    <li>convos</li>
    <li>perl530Packages.JSONValidator</li>
    <li>perl530Packages.LinkEmbedder</li>
    <li>perl530Packages.Minion</li>
    <li>perl530Packages.MinionBackendSQLite</li>
    <li>perl530Packages.MinionBackendmysql</li>
    <li>perl530Packages.MojoIOLoopDelay</li>
    <li>perl530Packages.MojoIOLoopForkCall</li>
    <li>perl530Packages.MojoJWT</li>
    <li>perl530Packages.MojoPg</li>
    <li>perl530Packages.MojoRedis</li>
    <li>perl530Packages.MojoSAML</li>
    <li>perl530Packages.MojoSQLite</li>
    <li>perl530Packages.MojoUserAgentCached</li>
    <li>perl530Packages.Mojolicious</li>
    <li>perl530Packages.MojoliciousPluginAssetPack</li>
    <li>perl530Packages.MojoliciousPluginGravatar</li>
    <li>perl530Packages.MojoliciousPluginMail</li>
    <li>perl530Packages.MojoliciousPluginOpenAPI</li>
    <li>perl530Packages.MojoliciousPluginStatus</li>
    <li>perl530Packages.MojoliciousPluginTextExceptions</li>
    <li>perl530Packages.MojoliciousPluginWebpack</li>
    <li>perl530Packages.Mojomysql</li>
    <li>perl530Packages.MusicBrainz</li>
    <li>perl530Packages.OpenAPIClient</li>
    <li>perl532Packages.JSONValidator</li>
    <li>perl532Packages.LinkEmbedder</li>
    <li>perl532Packages.Minion</li>
    <li>perl532Packages.MinionBackendSQLite</li>
    <li>perl532Packages.MinionBackendmysql</li>
    <li>perl532Packages.MojoIOLoopDelay</li>
    <li>perl532Packages.MojoIOLoopForkCall</li>
    <li>perl532Packages.MojoJWT</li>
    <li>perl532Packages.MojoPg</li>
    <li>perl532Packages.MojoRedis</li>
    <li>perl532Packages.MojoSAML</li>
    <li>perl532Packages.MojoSQLite</li>
    <li>perl532Packages.MojoUserAgentCached</li>
    <li>perl532Packages.Mojolicious</li>
    <li>perl532Packages.MojoliciousPluginAssetPack</li>
    <li>perl532Packages.MojoliciousPluginGravatar</li>
    <li>perl532Packages.MojoliciousPluginMail</li>
    <li>perl532Packages.MojoliciousPluginOpenAPI</li>
    <li>perl532Packages.MojoliciousPluginStatus</li>
    <li>perl532Packages.MojoliciousPluginTextExceptions</li>
    <li>perl532Packages.MojoliciousPluginWebpack</li>
    <li>perl532Packages.Mojomysql</li>
    <li>perl532Packages.MusicBrainz</li>
    <li>perl532Packages.OpenAPIClient</li>
  </ul>
</details>

**Comment by zakame** [comment]
Thanks :tada:

**Pull request merged**
